CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing Guidelines
+
+Thank you for taking the time to contribute to SheetShuttle! This guide will
+help you to effectively get started and contribute to the project.
+
+## Table of Contents
+
+- [Contributing Guidelines](#contributing-guidelines)
+  - [Table of Contents](#table-of-contents)
+  - [Raising an Issue](#raising-an-issue)
+  - [Making a Pull Request](#making-a-pull-request)
+  - [Project Standards](#project-standards)
+    - [Development Environment](#development-environment)
+    - [Automated Testing](#automated-testing)
+    - [Test Coverage](#test-coverage)
+    - [Code Linting](#code-linting)
+
+## Raising an Issue
+
+If you have a new issue to raise, first check the [Issue
+Tracker](https://github.com/GatorEducator/SheetShuttle/issues)
+to ensure someone has not already raised it. Select whether your
+issue is a bug report or a feature request. Please follow the provided
+template and use as much detail as possible when describing your issue.
+
```

Comment:
I think the bug report already has this but it would be nice to emphasize the need to provide steps to replicate if a bug is being reported. Also all the system information and the python version. It would be nice to have that here